slick2d-hiero/pom.xml

```diff
@@ -85,6 +90,22 @@
 					</execution>
 				</executions>
 			</plugin>
+			
+			<plugin>
```

Comment:
This seems repeated in the 3 different projects, should it be refactored inside the parent POM `pluginManagement` section and just inherited in the child projects?